metadata-ingestion/tests/integration/lookml/test_lookml.py

```diff
@@ -873,6 +873,7 @@ def test_view_to_view_lineage_and_liquid_template(pytestconfig, tmp_path, mock_t
             "_is_selected": True,
         },
         "source_region": "ap-south-1",
+        "star_award_winner_year": "public.winner_2025",
```

Comment:
is this also a liquid variable? or is it something else?

We should also add some docs around the liquid variable resolution functionality

Reply:
The `parameter` field, as described in the [Looker documentation](https://cloud.google.com/looker/docs/reference/param-field-parameter), is used to define variables accessible within Liquid templates using `{% ... %}` or `{{ ... }}` syntax. If no complex conditions need to be evaluated, these parameters can also be directly referenced in LookML views using the `@{ ... }` notation. 

For example:  
- `sql_table_name: @{star_award_winner_year}`  
- `sql_table_name: {% if user_attribute('use_schema') == 'true' %}
                      {{ star_award_winner_year }}
                   {% else %}
                      public.winner_2024
                   {% endif %} ;;`

The key difference between `@{ ... }` and using a Liquid template is that Liquid allows access to additional user-related attributes, while `@{ ... }` only references parameters defined in the `parameter` attribute of the view.

The attribute name `liquid_variable` in recipe  is bit confusing. We can rename it to `lookml_parameter`, then it wouldn't convey the dynamic nature of liquid template. I can add additional documentation to `liquid_variable` or you can suggest some other name

Reply:
This is actually incorrect - the `@{ variable }` syntax is for looker constants, which are defined in the manifest.lkml https://cloud.google.com/looker/docs/reference/param-manifest-constant?hl=en

The implementation here conflates parameters and constants in a way that is likely extremely confusing to the end user